tests/test_cli.py

```diff
@@ -8,6 +8,7 @@
 import click.testing
 import mock
 import pytest
+
```

Comment:
unnecessary change in this PR?

although it seems like maybe isort/black should have been failing here 🤔 

but can we open as a separate PR if it does need to be fixed?

Reply:
Isort says that this space "should be there", I move it to an other PR :+1: